apollo-federation/src/sources/connect/json_selection/README.md

```diff
@@ -99,7 +116,7 @@ JSArray              ::= "[" (JSLiteral ("," JSLiteral)*)? "]"
 StarSelection        ::= Alias? "*" SubSelection?
 NO_SPACE             ::= !SpacesOrComments
 SpacesOrComments     ::= (Spaces | Comment)+
-Spaces               ::= (" " | "\t" | "\r" | "\n")+
+Spaces               ::= ("⎵" | "\t" | "\r" | "\n")+
```

Comment:
The space was hard to see in [the railroad diagram](https://github.com/apollographql/router/blob/61566c7fa29fea5cb4b4ce53b1a87d9aa53be5fe/apollo-federation/src/sources/connect/json_selection/README.md#spaces-), since it didn't look like anything against the yellow background. I agree this text-based grammar with quotes should ideally use the actual space character, but I got lazy about making that change each time I rerendered the diagrams.